components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json

```diff
@@ -219,6 +219,24 @@
     "message": "Please backup your Brave wallet.",
     "description": "Description for backup wallet notification"
   },
+  "differentCustodianAccountLearnMore": {
```

Comment:
The name "differentCustodianAccount" seems to imply that this error happens when the *custodian* is different, rather than (what I think we mean) the custodian is the same but the *account* is different.

Also regarding naming, I think we should prefer "provider" as opposed to "custodian" in code.

Some alternatives might be:

* `alreadyLinkedWithWalletProvider`
* `walletProviderLimitReached` (similar to `deviceLimitReached`, but a wallet-provider specific limit of 1?)
* `providerLimitReached` (shortened version of above)

Based on how we obtain and use this information in the auth flow, I'm leaning toward the second, but what do you think?

Reply:
Yeah, it's sort of hard to come up with a good name for this one. 🙂 
This is not really a wallet provider limit though, it's an account limit when you're trying to link `>= 2` accounts from the same provider.
How about `mismatchedAccounts`?

Reply:
As per our Slack convo, we're going with `mismatchedProviderAccounts`.